src/components/Agencies/Agencies.css

```diff
@@ -42,14 +38,15 @@
 }
 .company-info h5 {
   margin-bottom: 5px;
+  overflow-wrap: break-word;
```

Comment:
This already fixes a lot

We need to dive deeper into the css of the card as a whole to fix it more fundementally